planner/scan.go

```diff
@@ -61,10 +61,16 @@ func (n *scanNode) Kind() string {
 }
 
 func (n *scanNode) Init() error {
+	acpModule, err := n.p.db.ACPModule(n.p.ctx)
```

Comment:
suggestion: There is no need for this to return an error. Other than the implementation on the transactional store, when you refactor it you could just return the acp module and call the function on line 73 directly.